WCSim.mac

```diff
@@ -107,15 +101,39 @@
 #/control/execute visOGLSX.mac
 #/control/execute visRayTracer.mac
 
+## select the input nuance-formatted vector file
+## you can of course use your own
+#/mygen/generator muline
+#/mygen/vecfile inputvectorfile
+#/mygen/vecfile h2o.2km.001-009x3_G4.kin
+#/mygen/vecfile mu+.out
 
 # Or you can use the G4 Particle Gun
-/mygen/generator normal
+# for a full list of /gun/ commands see:
+# http://geant4.web.cern.ch/geant4/G4UsersDocuments/UsersGuides/ForApplicationDeveloper/html/Control/UIcommands/_gun_.html
+/mygen/generator gun
 /gun/particle e-
 #/gun/particle pi0
 /gun/energy 500 MeV
 /gun/direction 1 0 0 
 /gun/position 0 0 0  
 
+# Or you can use the G4 General Particle Source
+# you can do a lot more with this than a monoenergetic, monodirectional, single-particle gun
+# for a full list of /gps/ commands see:
+# https://geant4.web.cern.ch/geant4/UserDocumentation/UsersGuides/ForApplicationDeveloper/html/ch02s07.html
+#/mygen/generator gps
+#/gps/particle e-
+#/gps/energy 500 MeV
+#/gps/direction 1 0 0
+#/gps/position 0 0 0
+
+# Or you can use the laser option
+# This is equivalent to the gps command, except that the gps particle energies are saved ignoring their mass
+# It is used for laser calibration simulation
+#/mygen/generator laser
```

Comment:
This is an improvement over the current WCSim.mac, which doesn't mention the laser option at all, but for completeness maybe we should include the commands to run laser like we have for the gps and gun above?